app/controllers/protocols_controller.rb

```diff
@@ -860,7 +860,8 @@ def load_organization_and_type
   def check_view_all_permissions
     load_organization_and_type
 
-    unless can_view_organization_protocols(@current_organization)
+    if @current_organization &&
+       !can_view_organization_protocols(@current_organization)
```

Comment:
Or just in single line:

render_403 unless can_view_organization_protocols(@current_organization)